docs/tutorial/snapcraft.md

```diff
@@ -91,14 +91,14 @@ version: '0.1'
 summary: Hello World Electron app
 description: |
   Simple Hello World Electron app as an example
-base: core18
+base: core22
 confinement: strict
 grade: stable
 
 apps:
   electron-packager-hello-world:
     command: electron-quick-start/electron-quick-start --no-sandbox
-    extensions: [gnome-3-34]
+    extensions: [gnome]
```

Comment:
Just making sure, is just `gnome` the extensions we want here, don't they version them (i.e. `gnome-3-38`)?: https://snapcraft.io/docs/gnome-3-38-extension

Reply:
Previous versions of GNOME extension had version suffixes, but [the new version that's compatible with core22](https://snapcraft.io/docs/supported-extensions) is simply named `gnome`.